cmake/windows/compilerconfig.cmake

```diff
@@ -42,9 +47,8 @@ if(CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION VERSION_LESS 10.0.20348)
                       "Please download and install the most recent Windows SDK.")
 endif()
 
-set(_obs_msvc_c_options /Brepro /MP /permissive- /Zc:__cplusplus /Zc:preprocessor)
-
-set(_obs_msvc_cpp_options /Brepro /MP /permissive- /Zc:__cplusplus /Zc:preprocessor)
+set(_obs_msvc_c_options /Brepro /MP)
+set(_obs_msvc_cpp_options /Brepro /MP)
```

Comment:
I believe clang-cl supports `/Brepro`.
https://blog.llvm.org/2019/11/deterministic-builds-with-clang-and-lld.html
https://releases.llvm.org/17.0.1/tools/clang/docs/UsersManual.html#id10